cli.js

```diff
@@ -21,6 +21,10 @@ if (!args.dir || (!args.all && (!args.platform || !args.arch))) {
   process.exit(1)
 }
 
+if (args.tmpdir === 'false') {
+  args.tmpdir = false
+}
```

Comment:
Yes, cannot be marked as "boolean". If you don't like this handling, maybe it will be better to add additional option `use-temp-dir` (default value: true) (but `tmpdir` and `use-temp-dir` are mutually exclusive, so, I think, it is better to allow specify `false`)?